policies/rudderc/tests/cases/general/escaping/metadata.xml

```diff
@@ -33,12 +33,10 @@
     </SECTION>
     <SECTION name="Technique parameters">
       <INPUT>
-        <NAME>3439BBB0-D8F1-4C43-95A9-0C56BFB8C27E</NAME>
-        <DESCRIPTION>server</DESCRIPTION>
-        <LONGDESCRIPTION>${sys.host} . | / ${sys.${host}} &apos; &apos;&apos; &apos;&apos;&apos; $ $$ &quot; &quot;&quot; \ \\😋aà3</LONGDESCRIPTION>
+        <NAME>server</NAME>
+        <DESCRIPTION>${sys.host} . | / ${sys.${host}} &apos; &apos;&apos; &apos;&apos;&apos; $ $$ &quot; &quot;&quot; \ \\😋aà3</DESCRIPTION>
         <CONSTRAINT>
-          <TYPE>textarea</TYPE>
-          <MAYBEEMPTY>true</MAYBEEMPTY>
+          <TYPE>string</TYPE>
```

Comment:
Why the change from textarea to string here?

Reply:
because previous code used to put textarea everywhere